elastic/filter/filter.go

```diff
@@ -0,0 +1,128 @@
+package filter
```

Comment:
I would call this indexer instead - since it's main purpose is indexing data into elasticsearch